gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/metastore/HiveMetaStoreUtils.java

```diff
@@ -256,6 +256,23 @@ public static SerDeInfo getSerDeInfo(HiveRegistrationUnit unit) {
     return si;
   }
 
+  /**
+   * Util for detecting if a table has a complex union (aka non-optional unions) column types. Iceberg currently does
```

Comment:
This is a simple utility method to tell if a hive table contains a complex union. Lets not complicate my mentioning iceberg.